src/view/terminalview.cpp

```diff
@@ -0,0 +1,34 @@
+#include "terminalview.h"
+
+std::string TerminalView::formatNumberSpacing(int number, int maxLen) {
+    std::string formattedString;
+    formattedString = std::to_string(number);
+    while (formattedString.length() < maxLen){
+        formattedString = " " + formattedString;
+    }
+    return formattedString;
+};
+
+std::string TerminalView::stringifyFood(Food food){
+    int xPos;
+    int yPos;
+    int value;
+
+    xPos = food.getPosition().x;
+    yPos = food.getPosition().y;
+    value = food.getValue();
+
+    std::string formattedXPos;
+    std::string formattedYPos;
+    std::string formattedValue;
+
+    formattedXPos = formatNumberSpacing(xPos, 4);
+    formattedYPos = formatNumberSpacing(yPos, 4);
+    formattedValue = formatNumberSpacing(value, 3);
+
+    std::string outputString;
+    outputString += "X: " + formattedXPos + " | ";
+    outputString += "Y: " + formattedYPos + " | ";
+    outputString += "Value: " + formattedValue + "\n";
+    return outputString;
```

Comment:
Sounds cool, I'll look into it

Reply:
I did some research and turns out it's best to use either += or .append() (they are pretty much the same under the hood)
Always avoid + since it creates a new array everytime.
<< should also be avoided unless you want to use to to concatenate with numbers it seems.
strcat() is outdated and slow

Look here for a quick reference https://www.thecodingdelight.com/string-cplusplus/

I will also write a research notion thing later